docs/crud/queries.md

```diff
@@ -4,4 +4,242 @@ title: Queries
+
+```graphql
+input NoteFilter {
+  id: IDInput
+  title: StringInput
+  opened: BooleanInput
+  likes: IntInput
+  completedPercentage: FloatInput
+  and: [NoteFilter!]
+  or: [NoteFilter!]
+  not: NoteFilter
+}
+```
+
+So you can perform filtering of the data like this:
+
+```graphql
+findNotes(filter: {
+  title: {
+    contains: "emails"
+  },
+  opened: {
+    eq: false
+  },
+  likes: {
+    gt: 10
+  },
+  completedPercentage: {
+    between: [20, 40]
+  },
+  and: {
+    title: {
+      startsWith: "read"
+    }
+  },
+  or: {
+    likes: {
+      eq: 100 
+    }
+  },
+  not: {
+    title: {
+      contains: "archived"
+    }
+  }
+ }) {
+  items {
+    id
+    title
+    likes
+  }
+}
+```
+
+#### Operators
+
+This is a list of all available filter operators.
+
+| Operator   	| Meaning                      	| Scalars    	|
+|------------	|------------------------------	|------------	|
+| eq         	| Equal to                     	| All        	|
+| ne         	| Not equal to                 	| All        	|
+| lt         	| Less than                    	| All        	| 
+| le         	| Less than or equal to        	| All        	| 
+| gt         	| Greater than                 	| All        	|  
+| gt         	| Greater than or equal to     	| All        	|  
+| between    	| Between a range (of numbers) 	| Int, Float 	|  
+| in         	| In a list of items           	| All        	|  
+| contains   	| String contains              	| String     	| 
+| startsWith 	| String starts with           	| String     	| 
+| endsWith   	| String ends with             	| String     	|
+
+Each field in the `NoteFilter` input maps to a generated scalar input. This specifies the operators available for that scalar.
+
+```graphql
+input StringInput {
+  ne: String
+  eq: String
+  le: String
+  lt: String
+  ge: String
+  gt: String
+  in: [String!]
+  contains: String
+  startsWith: String
+  endsWith: String
+}
+
+input IDInput {
+  ne: ID
+  eq: ID
+  le: ID
+  lt: ID
+  ge: ID
+  gt: ID
+  in: [ID!]
+}
+
+input IntInput {
+  ne: Int
+  eq: Int
+  le: Int
+  lt: Int
+  ge: Int
+  gt: Int
+  in: [Int!]
+  between: [Int!]
+}
+
+input FloatInput {
+  ne: Float
+  eq: Float
+  le: Float
+  lt: Float
+  ge: Float
+  gt: Float
+  in: [Float!]
+  between: [Float!]
+}
+
+input BooleanInput {
+  ne: Boolean
+  eq: Boolean
+}
+```
+
+### Pagination
+
+You can perform pagination with the `page` argument.
+
+```graphql
+type PageRequest {
+  limit: Int
+  offset: Int
+}
+```
+
+```graphql
+findNotes(page: {limit: 10, offset:4}) {
+  items {
+    id
+  }
+}
+```
+
+### Ordering
+
+```graphql
+input OrderByInput {
+  field: String!
+  order: SortDirectionEnum = ASC
+}
+
+enum SortDirectionEnum {
+  DESC
+  ASC
+}
+```
+
+To sort/order items by a certain field, use the `orderBy` argument.
+
+```graphql
+findNotes(orderBy: {order: DESC, field: "likes"}) {
+  items {
+    id
+  }
+}
+```
+
+## get
+
+### TODO
```

Comment:
This will be in the followup PR?

Reply:
Yes, the Queries page was large so wanted to get feedback on that early

Reply:
Good approach. I think we can review it separately once this lands.